src/parachain/vaults.ts

```diff
@@ -362,6 +370,18 @@ export class DefaultVaultsAPI {
         return encodeVault(vault, this.btcNetwork);
     }
 
+    async getLiquidationVaultId(): Promise<string> {
```

Comment:
Do you need that API since it does the same as the call below?

Reply:
It's more lightweight than the call below, so I thought it might be useful